Wikipedia/Code/NSUserDefaults+WMFExtensions.swift

```diff
@@ -116,17 +116,16 @@ let WMFSendUsageReports = "WMFSendUsageReports"
     
     @objc var themeAnalyticsName: String {
         let name = string(forKey: WMFAppThemeName)
-        let darkMode =  bool(forKey: "SystemDarkMode")
+        let systemDarkMode = systemDarkModeEnabled
         guard name != nil, name != Theme.defaultThemeName else {
-            let theme = Theme.defaultAnalyticsThemeName
-            return darkMode ? Theme.dark.name : theme
+            return systemDarkMode ? Theme.black.analyticsName : Theme.light.analyticsName
         }
         
         if Theme.withName(name)?.name == Theme.light.name {
             return Theme.defaultAnalyticsThemeName
         }
         
-        return Theme.withName(name)?.name ?? Theme.defaultAnalyticsThemeName
+        return Theme.withName(name)?.name ?? Theme.light.analyticsName
```

Comment:
@mazevedofs one tiny thing - the "-dimmed" name is still trickling through, I think this needs to be `Theme.withName(name)?.analyticsName` instead of `Theme.withName(name)?.name`.

Reply:
Thanks for catching this one as well! :)